pkg/pillar/cmd/msrv/msrv.go

```diff
@@ -0,0 +1,639 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Medatada server (msrv) creates an HTTP handler which is used by Zedrouter
+// to create server exposed to App Instances to get information, i.e. network
+// parameters and download patch envelopes.
+// You can think of metadata server as a translator from pubsub to REST API for
+// Application Instances
+
+package msrv
+
+import (
+	"bytes"
+	"context"
+	"encoding/gob"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/google/uuid"
+	"github.com/lf-edge/eve/pkg/pillar/agentbase"
+	"github.com/lf-edge/eve/pkg/pillar/base"
+	"github.com/lf-edge/eve/pkg/pillar/cipher"
+	"github.com/lf-edge/eve/pkg/pillar/flextimer"
+	"github.com/lf-edge/eve/pkg/pillar/persistcache"
+	"github.com/lf-edge/eve/pkg/pillar/pidfile"
+	"github.com/lf-edge/eve/pkg/pillar/pubsub"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/lf-edge/eve/pkg/pillar/utils/generics"
+	"github.com/lf-edge/eve/pkg/pillar/zedcloud"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	agentName    = "msrv"
+	errorTime    = 60 * time.Second
+	warningTime  = 40 * time.Second
+	stillRunTime = 25 * time.Second
+	// Publish 4X more often than zedagent publishes to controller
+	// to reduce effect of quantization errors
+	publishTickerDivider = 4
+)
+
+// Really a constant
+var nilUUID uuid.UUID
+
+// Version is set from the Makefile.
+var Version = "No version specified"
+
```

Comment:
We've removed this from all existing cmd/*/*.go so don't add it back here. (And I suspect it is not used.)

Reply:
Removed